lib/html.php

```diff
@@ -384,6 +384,95 @@ static public function error($message)
 		self::notice(l_t('Error'), $message);
 	}
 
+	/**
+	 * Renders reusable modal for tutorial mode, future help functionality
+	 * Breaks up message into array of slides based on HTML <br> element
+	 * 
+	 * @param string $page
+	 * @param string $message
+	 */
+	static public function help($page, $message)
+	{
+		if (isset($_COOKIE['wD-Tutorial'])) {

```

Comment:
bracketing inconsistent, next line please for most of the new brackets added in this file.